.github/CODEOWNERS

```diff
@@ -1,1427 +1,1852 @@
 # Instructions for CODEOWNERS file format and automatic build failure notifications:
+
 # https://github.com/Azure/azure-sdk/blob/main/docs/policies/opensource.md#codeowners
 
 ################
+
 # Automation
+
 ################
 
 # Git Hub integration and bot rules
+
 /.github/ @jsquire @ronniegeraghty
 
 ###########
+
 # SDK
+
 ###########
 
 # Catch all
+
 /sdk/ @xirzec
 
 # PRLabel: %Azure.Core
+
 /sdk/core/ @xirzec @jeremymeng @deyaaeldeen
 
 # PRLabel: %Azure.Core
+
 /sdk/core/core-amqp/ @jeremymeng @deyaaeldeen @HarshaNalluru
 
 # PRLabel: %Azure.Core
+
 /sdk/core/core-auth/ @xirzec @jeremymeng @deyaaeldeen
 
 # PRLabel: %Azure.Core
+
 /sdk/core/core-client-rest/ @xirzec @joheredi @jeremymeng @deyaaeldeen
 
 # PRLabel: %Azure.Core
+
 /sdk/core/core-tracing/ @xirzec @jeremymeng @joheredi
 
 # Service teams
+
 # PRLabel: %App Configuration
+
 /sdk/appconfiguration/ @HarshaNalluru
 
 # PRLabel: %Attestation
-/sdk/attestation/ @LarryOsterman @deyaaeldeen
+
+/sdk/attestation/ @LarryOsterman @deyaaeldeen @anilba06 @gkostal
```

Comment:
The core intended change looks good!

Reply:
@jsquire should we merged with those lines later in the same doc:

``` 
# ServiceLabel: %Attestation %Service Attention
#/<NotInRepo>/          @anilba06
```